app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -824,18 +824,18 @@ private void setupFilterPlaceholder(StatusViewData.Concrete status, StatusAction
 
         showFilteredPlaceholder(true);
 
-        String matchedKeyword = null;
+        Filter matchedFilter = null;
 
         for (FilterResult result : status.getActionable().getFiltered()) {
             Filter filter = result.getFilter();
             List<String> keywords = result.getKeywordMatches();
             if (filter.getAction() == Filter.Action.WARN && !keywords.isEmpty()) {
-                matchedKeyword = keywords.get(0);
+                matchedFilter = filter;
                 break;
```

Comment:
All of the keyword touching here can be removed if we're not using them

Reply:
Done.